binding/python/octopus.py

```diff
@@ -264,6 +265,8 @@ class CMatch(Structure):
             ("end_sec", c_float),
             ("probability", c_float)]
 
+    _PHRASE_REGEX = re.compile(r"^[a-zA-Z']+$")
```

Comment:
I think this will cause something like `search(metadata, ["search phrase one"]) to fail which I believe is a valid case and supported by the library.
```suggestion
    _PHRASE_REGEX = re.compile(r"^[a-zA-Z'\ ]+$")
```

Reply:
or
```suggestion
    _PHRASE_REGEX = re.compile(r"^\s*(?:\s?[A-Za-z])+\s*$")
```
which would accept any number of leading or trailing spaces, but would only allow a single space between words

Reply:
Octopus can handle multiple spaces between words (```      search     ...       phrase          ``` is equivalent to `search phrase`) so it depends how pedantic we want to be.